tests/README-containers.md

```diff
@@ -0,0 +1,134 @@
+# Testing ansible-freeipa modules with containers
+
+> This document describe the rationale, design and usage of `utils/ansible-freeipa-test.sh`.<br>
+> For the impatient, if you have [podman] installed, run `utils/ansible-freeipa-test.sh -m config` from the project root directory.
+
+
+## Introduction
+
+`ansible-freeipa` relies on two types of tests to verify the behavior is consistent with the modules design. There are pytest tests and Ansible playbook tests. All modules must provide a series of tasks in one or more Ansible playbook to test module behavior and idempotence verification, to be considered for inclusion.
+
+Each pull request triggers the execution of all registered tests, for all modules, given the environment is available, so it is possible to verify that the changes do not break existing behavior. The tests also run on different versions of FreeIPA, available for the latests versions of CentOS 7, CentOS 8, and Fedora.
+
+The problem is that, when executed in parallel, it takes about an hour to execute all tests in the current CI environment (as of May 2020), and it is not practical to wait so long to test minor modifications.
```

Comment:
Maybe it would be better to say:
As of May 2020 it takes about an our to ...  therefore it not practical to wait ....